templates/default/appserver.nginx.conf.erb

```diff
@@ -42,6 +42,18 @@ server {
     add_header X-Content-Type-Options nosniff;
   }
 
+<% if @out[:acao_packs] %>
+    location ^~ /packs/ {
+    gzip_static on;
+    expires max;
+    add_header Cache-Control public;
+    add_header Access-Control-Allow-Origin "$http_origin";
```

Comment:
Not at all familiar with Nginx configs, but it looks like this is reflecting the request origin (whatever the origin, it'll be sent back in the ACAO header). 

I've left that in for the moment, but that does create a discrepancy in behaviour between this and the Apache config where the default is the wildcard, and the user can provide a specific origin. 

Would like some guidance on what y'all think is the best approach.

Reply:
Honestly I'm not sure what webpack expects, but in theory - `Origin` should be fine here. Since packs are requested from the same server, they should be received from the origin anyway.